### ISSUE TITLE
create_swanTpar data for 2022 and latest

### ISSUE THREAD

**Issue by arief121** [jcwarner-usgs/COAWST#139]
Dear J,

I try to generate tpar data for do a swan simulation in December 2022. Is there any changes to get the data in 2022 using the script create_swanTpar_from_WW3.m? because in the script says that the data is only available from Feb 2005 to May 2019. 

Thank you for your help.

Arief

**Comment by jcwarner-usgs** [comment]
yes i see that the WW3 data is only available to 2019.
can you run a larger swan domain to get TPAR files?

**Comment by arief121** [comment]
yes, I can run a larger domain. any suggestions on where and how I get the TPAR files after 2019?

**Comment by jcwarner-usgs** [comment]
no i did not have time to persue this.
Please ask NOAA people, or others.  Ask on the WW3 forum, etc.  Dont just wait for me or this forum.
They might tell you to run ww3 global quick and get the bc's yourself.  you could look into that as well.
-j
